### ISSUE TITLE
Duckling Http Extractor Error on transforming

### ISSUE THREAD

**Issue by souvikg10** [RasaHQ/rasa_nlu#1021]
**Rasa NLU version**: 0.12

**Operating system** (windows, osx, ...): osx

**Content of model configuration file**:
```
language: "nl"

pipeline:
# this is using the spacy sklearn pipeline, adding duckling
# all components will use their default values
- name: "nlp_spacy"
- name: "tokenizer_spacy"
- name: "intent_featurizer_spacy"
- name: "ner_crf"
- name: "ner_synonyms"
- name: "intent_classifier_sklearn"
- name: "ner_duckling_http"
  locale: "nl"
  url: "http://duckling:8000"
- name: "intent_entity_featurizer_regex"
- name: "ner_spacy"
```

**Issue**:

I received the following error on calling duckling over http
```
rasa-nlu_1 | 2018-04-19 06:35:48+0000 [-] File "/app/rasa_nlu/rasa_nlu/extractors/duckling_extractor.py", line 53, in convert_duckling_format_to_rasa rasa-nlu_1 | 2018-04-19 06:35:48+0000 [-] "text": match["text"], rasa-nlu_1 | 2018-04-19 06:35:48+0000 [-] KeyError: u'text'
```

I did a bit of debugging and it seems at duckling_extractory.py at line 53 the extraction is

```
 "text": match["text"]
```

However the duckling is responding with this format
```
[{ "body": "32$", "start": 0, "value": { "value": 32, "type": "value", "unit": "$" }, "end": 3, "dim": "amount-of-money", "latent": false }]
```

I think it should be

```
"text":match["body"]
```

But this might break the existing Clojure version of duckling 

Can you maybe check?

**Comment by akelad** [comment]
just merged #1025 fixing this

**Comment by souvikg10** [comment]
Okay, I will retry and meanwhile I can close this. 

thanks
